src/components/accessibility/isAdmin.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Outlet, Navigate } from 'react-router-dom';
+
+function IsAdmin({role, loggedIn}) {
+  if (role !== "admin" && !loggedIn) return <Navigate to="login" />;
```

Comment:
- [ ] I think a better redirection for this `if` statement will be to be redirected to an "Unauthorized" page or to Home Page. Because by default users attempting to access this route might be because they know about its existence, normal users won't suffer any problem just by being redirected to Home Page

Reply:
You're right, I didn't even think about where best to route to, that's what happens when you just copy code right?! 🙈 Let's just redirect to `Home` for now and we could still decide to change that later. What do you think?

Reply:
Nice, thank you 👍